test_scripts/API/PerformAudioPassThru/028_ATF_TTS_IsReady_no_resp_PerfAudioPassThru_TTS_Speak_UNSUP_RES.lua

```diff
@@ -0,0 +1,172 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- [TTS Interface] HMI does NOT respond to IsReady and mobile app sends RPC that must be splitted
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one HMI-portions
+-- in this particular test it is checked case when TTS.Speak gets UNSUPPORTED_RESOURCE and UI.PerformAudioPassThru gets WARNINGS from HMI
+--
+-- 1. Used preconditions:
+-- HMI does not respond to UI.IsReady
+-- App is registered and activated SUCCESSFULLY
+-- 2. Performed steps:
+-- MOB -> SDL: sends PerformAudioPassThru
+-- HMI -> SDL: UI.PerformAudioPassThru (WARNINGS), TTS.Speak (UNSUPPORTED_RESOURCE),
+--
+-- Expected result:
+-- SDL -> HMI: resends TTS.Speak and UI.PerformAudioPassThru
+-- SDL -> MOB: PerformAudioPassThru (result code: UNSUPPORTED_RESOURCE, success: true)
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForUI_IsReady = require('user_modules/IsReady_Template/testCasesForUI_IsReady')
```

Comment:
@vvvakulenko, I think that reference here should be for testCasesForTTS_IsReady, also:
user_modules/IsReady_Template/testCasesForTTS_IsReady